setup.py

```diff
@@ -45,9 +45,9 @@
 setup_path = os.path.dirname(__file__)
 
 
-install_req = ['scipy>=0.17',
+install_req = ['scipy>=1.4.0',
```

Comment:
In which case, there is code in `model.py` and docs in `model.rst` which refer to LooseVersion of scipy >=1.2.0 for `dual_annealing` and `shgo` as options for `model.fit()`. This logic can be tidied up.

Reply:
Sounds good, would you like to take care of that in a PR to the main repository?

Reply:
Yep I'll do it later this morning

Reply:
Great, thanks!

Reply:
scipy 1.4 has been released in December 2019, which I think is a bit too recent. Scipy is not straightforward to package/distribute, so I would suggest to avoid increasing this requirement. As for numpy, in my opinion, it would be sensible to be conservative and as a guideline to supports scipy version released in the 2 years for example (see https://numpy.org/neps/nep-0029-deprecation_policy.html)

Reply:
The issue is that recent changes in ``decomposition`` require scipy v1.4.

Reply:
Which changes exactly? There have been a few PRs releted to decomposition, recently!

Reply:
The [arpack test](https://github.com/hyperspy/hyperspy/blob/81c226ac4fc6a32e09022ce0ce2b29bb90308269/hyperspy/tests/learn/test_svd_pca.py#L82) fails for scipy < 1.4

Reply:
Note that 1.4 also brings `scipy.fft` over `fftpack`. You can do a `pytest..importorskip("scipy", minversion="1.4")` for that test.

Reply:
It can definitely go to scipy 1.0.1 because that's what scikit-image requires (cf. #1438).

I won't make a separate PR in this case given @ericpre's comment.

Reply:
The thing is that, AFAIK, hyperspy is not available in any Linux distribution other than the ones with a rolling release policy such as Arch and [OpenSuse Tumbleweed](https://build.opensuse.org/package/show/home:andythe_great/python-hyperspy), for which requiring a recent scipy is not an issue. That's certainly because our dependencies, even when being conservative regarding scipy and numpy as currently, cannot be fulfilled in other distributions. Hopefully the situation will change with "The Split", but, until then, maybe it is not too bad to depend on a recent scipy?

Reply:
It seems that `svd_pca` will fail with `decomposition_output=None` and `svd_solver='arpack'.` but when using `decomposition`, the `output_dimension` is set (not None), so it should be fine to skip the failing test for this specific case?

Reply:
No the test for None is necessary, see https://github.com/hyperspy/hyperspy/blob/3ec3018f59cee4c9db023408b09d51024fe53d42/hyperspy/learn/svd_pca.py#L128

Reply:
I still think that scipy 1.4 is too recent. I will make a PR to raise an error when using arpack svd solver and scipy < 1.4.

Reply:
OK, we can go down to v1.1 that is 
already in Debian.

Reply:
#2471

Reply:
OK, I'll set scipy==1.0.1 as suggested by @tjof2